modules/top-level/output.nix

```diff
@@ -50,24 +50,49 @@ in
       default = false;
     };
 
-    finalPackage = mkOption {
-      type = types.package;
-      description = "Wrapped Neovim.";
-      readOnly = true;
-    };
+    build = {
+      # TODO: `standalonePackage`; i.e. package + printInitPackage + man-docs bundled together
 
-    initPath = mkOption {
-      type = types.str;
-      description = "The path to the `init.lua` file.";
-      readOnly = true;
-      visible = false;
-    };
+      package = mkOption {
+        type = types.package;
+        description = "Wrapped Neovim.";
+        readOnly = true;
+      };
+
+      initFile = mkOption {
+        type = types.path;
+        description = ''
+          The generated `init.lua` file.
+
+          > [!NOTE]
+          > If `performance.byteCompileLua` is enabled, this file may not contain human-readable lua source code.
+          > Consider using `build.initSource` instead.
+        '';
+        readOnly = true;
+        visible = false;
+      };
+
+      initSource = mkOption {
+        type = types.path;
+        description = ''
+          The generated `init.lua` source file.
 
-    printInitPackage = mkOption {
-      type = types.package;
-      description = "A tool to show the content of the generated `init.lua` file.";
-      readOnly = true;
-      visible = false;
+          This is usually identical to `build.initFile`, however if `performance.byteCompileLua` is enabled,
+          this option will refer to the un-compiled lua source file.
+        '';
+        readOnly = true;
+        visible = false;
+      };
+
+      printInit = mkOption {
```

Comment:
I don't know if I like this rename when reading through its usage in the tests and modules. Seems like its a change from our <name>Package usage everywhere for package options.

Reply:
Makes sense. If no one has any other suggestions I'll go with `build.printInitPackage` then.

Reply:
(Done)